### PR TITLE
[MM-26443] Implement loadtest reset command

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR adds a `loadtest reset` command to `ltctl` to help resetting the target instance's db and re-initialize the data.

#### Ticket

https://mattermost.atlassian.net/browse/MM-26443

### CODE REVIEW

File: cmd/ltctl/collect.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func RunCollectCmdF(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if output.HasAppServers() {
+	if !output.HasAppServers() {
```
Comment (streamer45) [inline cmd/ltctl/collect.go:154]: Unrelated fix.

File: cmd/ltctl/reset.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	agentClient, err := extAgent.NewClient(output.Agents.Value[0].PublicIP)
+	if err != nil {
+		return fmt.Errorf("error in getting ssh connection %w", err)
+	}
+
+	fmt.Println("Are you sure you want to delete everything? All data will be permanently deleted? [y/N]")
+	var confirm string
+	fmt.Scanln(&confirm)
+	if !regexp.MustCompile(`(?i)^(y|yes){1}?$`).MatchString(confirm) {
+		return nil
```
Comment (isacikgoz) [inline cmd/ltctl/reset.go:56]: We may want to return err here like we did [here](https://github.com/mattermost/mattermost-load-test-ng/blob/f02dfcc0a29f85cd0a3c6d697c2b6a896091466f/cmd/ltctl/report.go#L62). Or vice versa. 0/5 on returning error though. For instance `rm -i file` returns `0` when entered non-yes input.

Comment (streamer45) [reply cmd/ltctl/reset.go:56]: My idea is that if the answer is not yes the program should exit without errors (code 0). It's not an error to cancel the operation in my opinion.

Comment (isacikgoz) [reply cmd/ltctl/reset.go:56]: I see, I just wanted to point out we return errors in the other case. This is good to go as-is.

Comment (streamer45) [reply cmd/ltctl/reset.go:56]: Yes, I can update the other case for consistency :+1:
